about.html

```diff
@@ -13,6 +13,18 @@
 
 <body>
     <div class="container about-page">
+
+        <nav class="navBar">
+            <img src="img/lambda-black.png" alt="lambda logo">
```

Comment:
i don't believe there is anything incorrect with your code, but I was told that it may be semantically best to keep <img> tags out of the <nav>. unless the image is linked to allow the user to "navigate" elsewhere. ( you could do this by wrapping the <img> in an <a> tag with an href linked.) ..